docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,4 +1,4 @@
-# Onboarding Public Officers
+# Onboarding public officers using non-SE GSIB device
```

Comment:
let's standardise the unit of measure.
u have Onboarding public officers.. (plural)
and below is Onboarding vendor (singular)

perhaps can stick to singular unit of measure?

change to Onboarding public officer using non-SE GSIB device

Reply:
Ok noted the reason I may use plural is to avoid gender.  If gender text is not in the picture then I will use the singular form of the noun. Here I will change it to singular.

For example in the text following this heading, i have to use plural.